contracts/BondStorage.sol

```diff
@@ -62,6 +65,15 @@ contract BondStorage is AccessControl {
 
     mapping(address => BondRecord) issuedBonds;
 
+    function setBondingEvent(address _address) external onlyWhitelisted {
+        grantRole(WHITELIST_BOND_STORAGE, _address);
```

Comment:
ties back to my former question: do we want to give out permissions through the `BondStorage` contract or through the operator? I could see the benefits of letting the operator have more control over handling the privileges of multiple contracts which means to reduce this contract's own power. (Also for simplicity of use.)

Reply:
i suppose we'd have to add the operator to the deployment parameters, so we can give the operator admin roles on initialisation, but i think that would be possible